### ISSUE TITLE
Yoast.js crashed if there's carriage return in an HTML tag

### ISSUE THREAD

**Issue by stephanrauh** [Yoast/YoastSEO.js#1073]
## What did you expect to happen?
I expected to see the result of the Flesch reading test.

## What happened instead?
Yoast.js throws an error:
```
wp-seo-post-scraper-421.min.js?ver=4.2.1:10 Error: unable to tokenize
    at Object.end (wp-seo-post-scraper-421.min.js?ver=4.2.1:6)
    at j (wp-seo-post-scraper-421.min.js?ver=4.2.1:16)
    at p (wp-seo-post-scraper-421.min.js?ver=4.2.1:16)
    at c (wp-seo-post-scraper-421.min.js?ver=4.2.1:6)
    at d (wp-seo-post-scraper-421.min.js?ver=4.2.1:4)
    at d (wp-seo-post-scraper-421.min.js?ver=4.2.1:6)
    at d (wp-seo-post-scraper-421.min.js?ver=4.2.1:5)
    at b.exports (wp-seo-post-scraper-421.min.js?ver=4.2.1:16)
    at Object.b.exports [as getSentenceBeginnings] (wp-seo-post-scraper-421.min.js?ver=4.2.1:15)
    at J.getResearch (wp-seo-post-scraper-421.min.js?ver=4.2.1:11)
```
## How can we reproduce this behavior?
Create an empty blog Wordpress blog post and enter an image tag with a line feed:
![image](https://cloud.githubusercontent.com/assets/3045767/23094648/4aa0d05e-f5fd-11e6-8762-ab4c8f8199b0.png)

## Technical info

* Platform: WordPress
* Platform version: 4.7.2
* Yoast SEO version: 4.2.1

**Comment by CarolineGeven** [comment]
Thanks for reporting this issue. We've been able to reproduce the issue and have therefore labeled the issue as a bug.

Unfortunately we're not able to fix this at short notice. Currently we're focusing on issues that affect many users. We've concluded that this issue is not experienced by many users, therefore we are not able to fix this short-term. However, if more users are affected by this bug, we'll of course revisit this issue.

**Comment by stephanrauh** [comment]
May providing a pull request speed up things?

**Comment by atimmer** [comment]
@stephanrauh Yes, pull requests are very welcome 👍🏻

**Comment by monbauza** [comment]
Please inform the customer of conversation # 262496 when this conversation has been closed.

**Comment by benvaassen** [comment]
This should be fixed with 9.2 (see #840).
